### ISSUE TITLE
Describe the implications of IMPORT_FORCE=/ --import.force true vs. false

### ISSUE THREAD

**Issue by pmorch** [adorsys/keycloak-config-cli#356]
`README.md` doesn't really say much about these two except:

> Enable force import of realm config

:-)

`de.adorsys.keycloak.config.service.RealmImportService#updateRealmIfNecessary` has:

```java
    private void updateRealmIfNecessary(RealmImport realmImport) {
        if (importProperties.isForce() || checksumService.hasToBeUpdated(realmImport)) {
            updateRealm(realmImport);
        } else {
            logger.debug(
                    "No need to update realm '{}', import checksum same: '{}'",
                    realmImport.getRealm(),
                    realmImport.getChecksum()
            );
        }
    }
```

So I'm guessing that if the input is unchanged since last run, the realm is left as-is, but if anything in the input has changed, the checksum is different, and everything is applied. Is that correct?

**Comment by jkroepke** [comment]
Correct, thats all. Feel free to provide an understandable description as PR.

**Comment by jkroepke** [comment]
Resolved by #357
